src/main/java/by/custom/utilcalculator/service/MessagesCreator.java

```diff
@@ -35,43 +33,17 @@ public static MessagesCreator getInstance() {
 
     //this method builds next questions for user to interact with chatbot
     public String buildNextStepQuestion(final UserProgress userProgress) throws UtilsborCommandTreeReadingException {
-        final Step step = userProgress.getNextStep();
-        switch (step) {
-            case GENERAL_TRANSPORT_TYPE -> {
-                return getGreeting();
-            }
-            case PARTICULAR_TRANSPORT_TYPE -> {
-                return getTransportType(userProgress);
-            }
-            case WEIGHT -> {
-                return getTransportWeight(userProgress);
-            }
-            case ENGINE_TYPE -> {
-                return getTypeOfEngine();
-            }
-            case ENGINE_VOLUME_POWER -> {
-                return getEngineVolumeOrPower(userProgress);
-            }
-            case COUNTRY_ORIGIN -> {
-                return getCountryOrigin();
-            }
-            case OWNERS_TYPE -> {
-                return getTypeOfOwner();
-            }
-            case TRUCK_UNIT_CLASS -> {
-                return getTruckUnitClass();
-            }
-            case TRAILERS_O4_TYPE -> {
-                return getO4TrailersTypes();
-            }
-            case AGE -> {
-                return getAgeAuto();
-            }
-            case FAREWELL -> {
-                return getResultAndFarewell(userProgress);
-            }
+        String nextMessage;
+        try {
+            nextMessage = bundle.getString(userProgress.getNextMessage());
+        } catch (NullPointerException e) {
```

Comment:
What would happen if another exception were thrown? Maybe better to catch more general exception?
```suggestion
        } catch (Exception e) {
```

Reply:
Done, but I would like to ask you about  explanations. 
We are requesting a string from a bundle and we can receive this string (if it exists in bundle) or we can not receive a string (if there is not such string in bundle). In my opinion another exceptions here are redundant - the only possible exception here is the NPE.

Reply:
If you are sure that only NPE is possible, you can leave NullPointerException of course.